MBBSEmu/CPU/CPUCore.cs

```diff
@@ -2671,6 +2708,11 @@ private void Op_Call()
                                 Memory.GetWord(Registers.GetValue(_currentInstruction.MemorySegment), offset);
                             Registers.IP = destinationOffset;
 
+#if DEBUG
```

Comment:
all this debugging code necessary anymore?

Reply:
Yes, when debugging CPU/ASM issues. In addition to being within an preprocessor it'll output when `_showDebug` is true, which is only used for outputting ASM for debugging.